### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by ketan**

### CODE REVIEW

File: server/src/com/thoughtworks/go/config/GoConfigFileDao.java [hunk 1]
```diff
@@ -177,36 +141,6 @@ public String md5OfConfigFile() {
         return cachedConfigService.currentConfig().getMd5();
     }
 
-    public void updatePipelineGroup(final PipelineConfigs pipelineConfigs, final String groupName, final String md5) {
```
Comment (zabil) [inline server/src/com/thoughtworks/go/config/GoConfigFileDao.java]: You can edit the pipeline group through the ui when the admin sets pipeline group permissions. Are you sure this API is not called?

Comment (ketan) [inline server/src/com/thoughtworks/go/config/GoConfigFileDao.java]: nope, for every method that is removed, I executed this code -

```
git grep -i updatePipelineGroup && git grep 'update_pipeline_group'
```

**Comment by srinivasupadhya** [comment]
@zabil reviewed this. accepting on his behalf.

**Comment by zabil** [comment]
The GoConfigAdministrationController routes all saves through "updateConfig" in the dao.

**Pull request merged**